doc/dev/CredscanProcess.md

```diff
@@ -0,0 +1,66 @@
+# Guide for Validating the Build Compliance check on azure-sdk-for-net PRs
+
+This guide describes how package owners can monitor their package's Credential Scanner (CredScan) status and correct any errors.
+General information about CredScan can be found in the overview documentation at [aka.ms/credscan](credscan_doc). The
+Azure SDK's motivation and methodology for running CredScan is documented [here](devops_doc).
+
+## Table of Contents
+- [Check CredScan Status](#check-credscan-status)
+- [Correct Active Warnings](#correct-active-warnings)
+  - [True Positives](#true-positives)
+  - [False Positives](#false-positives)
+- [Support](#support)
+
+## Check CredScan Status
+If your PR fails the Build Compliance check, follow these steps: 
+
+1. Click “View more details on Azure Pipelines”. In the “Compliance” check on the Pipeline page, view the “Scanning for credentials” step. This page will show where the leaked secrets are in the current PR.
+
+2. Each warning will begin with the path to the file containing a potential credential, as well as the row and column where
+the credential string begins. For example, for a potential credential that starts in row 3 and column 20 of a
+particular file:
+```
+##[error]sdk/{service}/{package}/{file}.json:sdk/{service}/{package}/{file}.json(3,20)
+```
+
+The warning will then list an error code and description of why the potential credential was flagged.
+
+## True Positives
```

Comment:
We have a general guidance on how we mitigate the issue. I have a link in wiki: aka.ms/credscan. Better to include the information there, as we need to rotate the secret besides of sanitizer.

Reply:
I think the aka.ms link is in the first paragraph so hopefully that is ok, moving other information to internal

Reply:
aka.ms/credscan did not say anything about rotate the secret.

This is the link talking about this: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/101/Leaked-secret-procedure

Reply:
I copied over the leaked secret procedure into this doc (I wasn't sure if I could link to Internal link) so hopefully it's ok now.